lib/jira/jwt_client.rb

```diff
@@ -14,6 +15,30 @@ def make_multipart_request(url, data, headers = {})
       super(url, data, headers)
     end
 
+    class JwtBuilder
+      attr_reader :request_url, :http_method, :shared_secret, :site, :issuer
+
+      def initialize(request_url, http_method, shared_secret, site, issuer)
+        @request_url = request_url
+        @http_method = http_method
+        @shared_secret = shared_secret
+        @site = site
+        @issuer = issuer
+      end
+
+      def build
+        claim = Atlassian::Jwt.build_claims \
+          issuer,
+          request_url,
+          http_method.to_s,
+          site,
+          (Time.now - 60).to_i,
+          (Time.now + 86_400).to_i
+
+        JWT.encode claim, shared_secret
+      end
+    end
+
     class JwtUriBuilder
```

Comment:
Is it used somewhere outside of this file?
The only usage in this file is removed `JwtUriBuilder`